lib/tasks/export_local_transactions.rake

```diff
@@ -0,0 +1,15 @@
+desc "Export Local Transactions' slugs, names, LGSL and optional LGIL codes as CSV"
+
+task :export_local_transactions => :environment do
+  require "csv"
+
+  csv_string = CSV.generate do |csv|
+    csv << ["slug","lgsl","lgil","title"]
+
+    Edition.where(_type: "LocalTransactionEdition").each do |lte|
+      csv << [lte.slug,lte.lgsl_code,lte.lgil_override,lte.title]
```

Comment:
This is mainly for Mark Sheldon and the Custom team's use in our work around local DirectGov and transactions, as far as I know. I only made it into a rake task in case we needed to run it more than once easily.